PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,9 +3,9 @@ Pull request title
 
 The format of the Pull request title must be:
 
-    <component>: <short_change_message> #<issue number>
+    <short_change_message> #<issue number>
```

Comment:
again, i like to have the component in there. the labels for components only show up on github, but never in the gitlog

Reply:
What the tools/submit-pull-request does is it replicates the commit message as the body of the PR, so it also includes the <component>.

Reply:
are the duplicated component names remove from the release notes then ? If yes, we can let the component here

Reply:
The release notes are made from the issues, not from the commit messages. So in the issues they should not be mentioned, as we have the labels there (Also for search-ability). In the commit messages it is still nice to have them I think, as one might want to go through git log and easily see which components are involved.

Reply:
fixed